### DISCUSSION TITLE
[Cancelled] WinUI Community Call: July 20, 2022 ☀️

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/microsoft-ui-xaml#7377] [Announcements]
We have decided to cancel the July edition of the WinUI Community Call as many folks on the WinUI team are out of office and enjoying the (Northern hemisphere's) summer weather.

For any questions you have, you can:
- Open a Q&A [discussion](https://github.com/microsoft/microsoft-ui-xaml/discussions/new?category=q-a&title=%5BYour+Question+Here%5D) for members of the community or WinUI team to answer.
- Leave a comment on the August community call discussion for us to answer live on August 17th: #7375

News for July & August:
- Keep your eyes out for WinUI 2.8, Windows App SDK 1.2-experimental, and Windows App SDK servicing releases.
- At the June call, we noted that we planned to discuss the 1.2 feature list at the July call, this is still in the works and will be published when available.
- If you are interested in C# Windows UI content on July 20th, @michael-hawker will be hosting a talk about the [Windows Community Toolkit](https://github.com/CommunityToolkit/WindowsCommunityToolkit) (WCT) - Toolkit Labs at the [.NET Foundation](https://dotnetfoundation.org/) Summit (DNF) at 11:30 (PT) | 14:30 (ET) |  18:30 (UTC). 
  - Join live at: https://www.dnfsummit.org/

We look forward to hearing from you at the next call!

**Comment by michael-hawker** [comment]
Thanks Gabby for calling out the talk!

I'll be talking about our new [Toolkit Labs for Windows](https://aka.ms/toolkit/labs/windows) project we've been working on and how it'll help us build the Windows Community Toolkit for UWP and the Windows App SDK moving forward. 🎉

**Reply by gabbybilka** [reply]
Looking forward to attending! 😊
